app/move/fields/create.js

```diff
@@ -344,17 +344,44 @@ module.exports = {
   },
   date_from: {
     ...dateField,
-    validate: [...dateField.validate, 'required', 'after'],
+    validate: [...dateField.validate, 'after'],
     id: 'date_from',
     name: 'date_from',
     label: {
       text: 'fields::date_from.label',
       classes: 'govuk-label--s',
     },
   },
+  has_date_to: {
+    validate: 'required',
+    component: 'govukRadios',
+    name: 'has_date_to',
+    fieldset: {
+      legend: {
+        text: 'fields::has_date_to.legend',
+        classes: 'govuk-label--m',
+      },
+    },
+    items: [
+      {
+        value: 'yes',
+        text: 'Yes',
+        conditional: 'date_to',
+      },
+      {
+        value: 'no',
+        text: 'No',
+      },
+    ],
+  },
   date_to: {
+    skip: true,
```

Comment:
Ideally any properties just for this step should be defined after the shared object which is included.